### DISCUSSION TITLE
Can use supabase auth with payloadcms?

### DISCUSSION THREAD

**Discussion by gor3a** [payloadcms/payload#9167] [Q&A]
I want to integrate with supabase auth module and payloadcms, I want supabase to manage all non-functional (rate limit) and also manage SSO with supabase and OTP, also I want to make the users collection align with the status of every user on supabase also RBAC to create (admin, supplier, editor, customer), is there anyone has try it?

**Comment by Kamenow** [comment]
@gor3a Have you managed to get this setup?

**Reply by gor3a** [reply]
@Kamenow No I didn't I pause the flow to integrate Supabase Auth with PayloadCMS, and go for another flow
